File: C/ex17.c

#include <stdio.h>
#include <stdlib.h>



int main(int argc, char *argv[])

{
int catetooposto, catetoadjacente, hipotenusa,cat,hipot;

printf("Digite a medida do catetooposto:  ");
scanf("%i", &catetooposto);

printf("Digite a medida do catetoadjacente:  ");
scanf("%i", &catetoadjacente);

printf("Digite a medida a hipotenusa:  ");
scanf("%i", &hipotenusa);

cat=((catetooposto*catetooposto)+(catetoadjacente*catetoadjacente));
hipot=(hipotenusa*hipotenusa);

if(cat==hipot){
printf("Triangulo Retangulo!");
}

else{
        printf("Nao e Triangulo Retangulo!");
    }
    
 
 return 0;

}

File: PYTHON/Ex17.py


a=float(input ('digite o valor do cateto oposto:     '))
b=float(input ('digite o valor do cateto adjcente:   '))
c=float(input ('digite o valor da hipotenusa:        '))

if((c*c)==(a*a)+(b*b)):
    print(f" é tringulo retangulo")
else:
    print(f"não é trinagulo retangulo")
